pkg/operator/status.go

```diff
@@ -280,6 +280,15 @@ func (optr *Operator) syncUpgradeableStatus(co *configv1.ClusterOperator) error
 		Reason: asExpectedReason,
 	}
 
+	configNode, err := optr.configClient.ConfigV1().Nodes().Get(context.Background(), ctrlcommon.ClusterNodeInstanceName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	if configNode.Spec.CgroupMode == configv1.CgroupModeV1 {
+		coStatusCondition.Status = configv1.ConditionFalse
+		coStatusCondition.Reason = "ClusterOnCgroupV1"
+		coStatusCondition.Message = "Cluster is using cgroup v1 and is not upgradable. Please update the `CgroupMode` in the `nodes.config.openshift.io` object to 'v2'. Once upgraded, the cluster cannot be changed back to cgroup v1"
```

Comment:
For the message, perhaps it would be good to mention the deprecation of v1 as well, just to clarify why it's not upgradeable (does it make sense to link to anything here?)

Reply:
I have added `deprecated cgroup v1` in the message.
We already have the deprecation of cgroup v1 message in the config node object's status condition for the last few releases.
We have a reference of OCPSTRAT for this feature as of now. We don't have a concrete link to when exactly the RHCOS would be removing the support for cgroup v1 i.e. it's going to be done in the future releases (may be by 4.20?). So, I don't think we have a link at the moment to convey in the message.